calculator-server/controllers/calculator-stateless.js

```diff
@@ -0,0 +1,67 @@
+const Calculator = require("../../core/calculator");
+
+/*
+{
+    “operand1”: 1;
+    “operand2”: 24;
+    “operation”: “add”;
+}
+*/
+const answer = (req, res) => {
+  console.log(req.body);
+  const { operand1, operand2, operation } = req.body;
+
+  const calculator = new Calculator();
+  calculator.add(operand1);
+
+if (operation === undefined){
+  res.status(400).json({ error: "An operation is needed" });
+}
+
+if ((typeof operand1 !== 'number' )|| (typeof operand2 !== 'number' ) ){
+  res.status(400).json({ error: "Please enter numbers as the operands" });
+}
+
+if (!(operand1 && operand2)){
```

Comment:
If an operand was not provided, then of course it's not going to be a number, as per the above validation, so this validation needs to be done before that one

Reply:
True, fixed that